Lib/fontTools/feaLib/builder.py

```diff
@@ -508,16 +508,10 @@ def setElidedFallbackName(self, names, location):
                     f"{token} is already set.",
                     location,
                 )
-        self.stat_["ElidedFallbackName"] = names
-
-    def setElidedFallbackNameID(self, value, location):
-        for token in ("ElidedFallbackName", "ElidedFallbackNameID"):
-            if token in self.stat_:
-                raise FeatureLibError(
-                    f"{token} is already set.",
-                    location,
-                )
-        self.stat_["ElidedFallbackNameID"] = value
+        if isinstance(value, int):
+            self.stat_["ElidedFallbackNameID"] = value
+        if isinstance(value, list):
+            self.stat_["ElidedFallbackName"] = value
```

Comment:
even if unlikely it's always worth adding a final else block for self-protection  
```
else:
    raise AssertionError(value)
```

Reply:
Also, that second if should be an elif.

Reply:
Updated